common/src/main/java/tachyon/network/protocol/RPCMessage.java

```diff
@@ -50,6 +50,15 @@ public void encode(ByteBuf out) {
     }
 
     /**
+     * Returns the int identifier of the type.
+     *
+     * @return the int id of the type
+     */
+    public int getId() {
+      return mId;
```

Comment:
I changed DataServerMessage.mRPCMessageType to be Type. I also added some comments to clarify the situation.

The main reason this is sort of messy is that DataServerMessage still encodes and decodes everything in a manual way. Once everything uses netty, it should be cleaner.